File: 03.business-os/NameCardManager/iphone/NameCardManager/BusinessOSCommonOSLaunch/BusinessOSCommonOSLaunchRegistry.swift
import Foundation

enum BusinessOSCommonOSLaunchRegistry {
    static func createDefault() -> BusinessOSCommonOSLaunchDescriptor {
        BusinessOSCommonOSLaunchDescriptor(
            appName: "NameCardManager",
            launchMode: "launch_via_commonos_consumer",
            defaultTarget: "businessos-commonos-consumer",
            consumerTarget: "_commonos",
            providerRole: "shared_provider",
            consumerRole: "os_side_consumer",
            uiOwner: "CommonOS",
            businessOwner: "NameCardManager"
        )
    }
}

File: 03.business-os/NameCardManager/android/app/src/main/java/com/lsam/NameCardManager/launch/BusinessOSCommonOSLaunchRegistry.java
package com.lsam.NameCardManager.launch;

public final class BusinessOSCommonOSLaunchRegistry {

    public BusinessOSCommonOSLaunchDescriptor createDefault() {
        return new BusinessOSCommonOSLaunchDescriptor(
                "NameCardManager",
                "launch_via_commonos_consumer",
                "businessos-commonos-consumer",
                "_commonos",
                "shared_provider",
                "os_side_consumer",
                "CommonOS",
                "NameCardManager"
        );
    }
}
